lib/tests/formulae_dependents.rb

```diff
@@ -54,6 +66,21 @@ def dependent_formulae!(formula_name, args:)
       end
 
       def dependents_for_formula(formula, formula_name, args:)
+        # Calling `brew uses` is slow, so let's reserve doing that for
+        # if we can find a formula in the same tap that declares a dependency
+        # on the formula we are testing.
+        has_dependents = formula.tap.potential_formula_dirs.any? do |dir|
+          next false unless dir.exist?
+
+          dir.children.any? do |child|
+            next false unless child.file?
+            next false unless child.extname == ".rb"
+
+            child.read.include? "depends_on \"#{formula_name}\""
```

Comment:
I'd really love to avoid something this hacky as this when we have RuboCop AST methods to do this sort of thing.

Reply:
Can we construct the AST without loading the formula? Because that's part of what's slow here.

Reply:
For example, doing this instead:
```ruby
        has_dependents = formula.tap.formula_names.any? do |name|
          Formula[name].deps.map(&:name).include? formula_name
        end
```
We get:
```
( brew test-bot --only-formulae-dependents --testing-formulae=hello --dry-run)  12.84s user 2.96s system 68% cpu 23.128 total
```
It's faster than just doing `brew uses`, but also a about 6 times slower than just matching text.